shared/lib_battery_dispatch.h

```diff
@@ -152,8 +149,8 @@ class dispatch_t
 
 	/**
 	The dispatch mode.
-	For behind-the-meter dispatch: 0 = LOOK_AHEAD, 1 = LOOK_BEHIND, 2 = MAINTAIN_TARGET, 3 = CUSTOM, 4 = MANUAL, 5 = RESILIENCE
-	For front-of-meter dispatch: 0 = LOOK_AHEAD, 1 = LOOK_BEHIND, 2 = INPUT FORECAST, 3 = CUSTOM, 4 = MANUAL, 5 = RESILIENCE
+	For behind-the-meter dispatch: 0 = LOOK_AHEAD, 1 = LOOK_BEHIND, 2 = MAINTAIN_TARGET, 3 = CUSTOM, 4 = MANUAL, 5 = FORECAST, 6 = RESILIENCE
```

Comment:
Looks like all enums were updated...

Reply:
The front of meter enums weren't changed (see line 47), this updates the documentation to match the actual enum names.